modules/distr-tracing-tempo-install-tempostack-cli.adoc

```diff
@@ -0,0 +1,169 @@
+// Module included in the following assemblies:
+//
+//* observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-install-tempostack-cli_{context}"]
+= Installing a TempoStack instance by using the CLI
+
+You can install a TempoStack instance from the command line.
+
+.Prerequisites
+
+* An active {oc-first} session by a cluster administrator with the `cluster-admin` role.
++
+[TIP]
+====
+* Ensure that your {oc-first} version is up to date and matches your {product-title} version.
+
+* Run `oc login`:
++
+[source,terminal]
+----
+$ oc login --username=<your_username>
+----
+====
+
+* You have completed setting up the required object storage by a supported provider: link:https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation[Red Hat OpenShift Data Foundation], link:https://min.io/[MinIO], link:https://aws.amazon.com/s3/[Amazon S3], link:https://azure.microsoft.com/en-us/products/storage/blobs/[Azure Blob Storage], link:https://cloud.google.com/storage/[Google Cloud Storage]. For more information, see "Object storage setup".
++
+[WARNING]
+====
+Object storage is required and not included with the {TempoShortName}. You must choose and set up object storage by a supported provider before installing the {TempoShortName}.
+====
+
+.Procedure
+
+. Run the following command to create a project of your choice for the TempoStack instance that you will create in a subsequent step:
++
+[source,terminal]
+----
+$ oc apply -f - << EOF
+apiVersion: project.openshift.io/v1
+kind: Project
+metadata:
+  name: <project_of_tempostack_instance>
+EOF
+----
+
+. In the project that you created for the TempoStack instance, create a secret for your object storage bucket by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f - << EOF
+<object_storage_secret>
+EOF
+----
++
+For more information, see "Object storage setup".
++
+--
+include::snippets/distr-tracing-tempo-secret-example.adoc[]
+--
+
+. Create a TempoStack instance in the project that you created for it.
++
+[NOTE]
+====
+You can create multiple TempoStack instances in separate projects on the same cluster.
+====
++
+.. Customize the `TempoStack` custom resource (CR):
++
+[source,yaml]
+----
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: sample
+  namespace: <project_of_tempostack_instance>
+spec:
+  storageSize: 1Gi
+  storage:
+      secret: # <1>
+          name: <secret-name> # <2>
+          type: <secret-provider> # <3>
+  template:
+    queryFrontend:
+      jaegerQuery:
+        enabled: true
+        ingress:
+          route:
+            termination: edge
+          type: route
+----
+<1> The secret you created in step 2 for the object storage that had been set up as one of the prerequisites.
```

Comment:
Could this be:
```suggestion
<1> The secret you created in the previous step for the object storage that had been set up as one of the prerequisites.
```
? Just so you don't have to worry about the numbering changing later. If so, same for all other instances of "step 2" in PR.

Reply:
:slightly_smiling_face: thank you for being considerate, but I'm keeping the numbers on purpose here so that it won't accidentally start to refer to the wrong step if they add or reorganize steps (both of which happened in the past).